# https://github.com/numpy/numpy/issues/27699

In file included from ../numpy/_core/src/common/npy_cpu_dispatch.h:41,
                 from ../numpy/_core/src/common/npy_config.h:5,
                 from ../numpy/_core/include/numpy/npy_common.h:10,
                 from ../numpy/_core/src/common/simd/simd.h:12,
                 from ../numpy/_core/src/umath/loops_autovec.dispatch.c.src:12:
../numpy/_core/src/umath/loops_autovec.dispatch.c.src: In function 'ULONG_bitwise_count':
../numpy/_core/src/umath/loops_autovec.dispatch.c.src:108:43: internal compiler error: Segmentation fault
 NPY_NO_EXPORT void NPY_CPU_DISPATCH_CURFX(@TYPE@_bitwise_count)
                                           ^~~~~~~~~~~~~~~~~~~
---

--- a/numpy/_core/src/umath/loops_autovec.dispatch.c.src
+++ b/numpy/_core/src/umath/loops_autovec.dispatch.c.src
@@ -104,11 +104,13 @@ NPY_NO_EXPORT void NPY_CPU_DISPATCH_CURF
 #endif
 }
 
+#pragma GCC optimize ("0") // Deactivate optimisations
 NPY_NO_EXPORT void NPY_CPU_DISPATCH_CURFX(@TYPE@_bitwise_count)
 (char **args, npy_intp const *dimensions, npy_intp const *steps, void *NPY_UNUSED(func))
 {
     UNARY_LOOP_FAST(@type@, npy_ubyte, *out = npy_popcount@c@(in));
 }
+#pragma GCC reset_options // Re-activate optimisations
 
 /**end repeat**/
 
